.travis.yml

```diff
@@ -8,13 +15,18 @@ branches:
     - develop
     - master
 
+env:
+  - ANSIBLE_INSTALL_VERSION=1.9.6
+  - ANSIBLE_INSTALL_VERSION=2.0.0.2
```

Comment:
Ansible 2.0? Are you sure? Who will be able to support it? Who will be able to reuse it?